### PR TITLE
Feat: components cleanup [TAS-71]

### PR DESCRIPTION
**Pull request by femacedogomes**

### CODE REVIEW

File: src/components/atoms/Navigation/NavMobile.tsx [hunk 1]
```diff
@@ -122,9 +28,7 @@ const NavMobile: React.FC<NavMobileProps> = ({
           <ButtonClose onClick={onClickClose} />
         </span>
       </div>
-      <ul className="flex flex-col py-6 px-2 space-y-1">
-        {data.map(_renderItem)}
-      </ul>
+      <ul className="flex flex-col py-6 px-2 space-y-1"></ul>
```
Comment (renanpontez) [inline src/components/atoms/Navigation/NavMobile.tsx:31]: Why `{data.map(_renderItem)}` was removed?

Comment (renanpontez) [reply src/components/atoms/Navigation/NavMobile.tsx:31]: Will we still need this line?

File: src/components/mols/index.ts [hunk 2]
```diff
@@ -7,6 +6,8 @@ export { default as CouponActivatedPage } from "./CouponActived";
 export { default as CouponActived } from "./CouponActived";
 export { default as UserCoupons } from "./UserCoupons";
 export { default as FastLoginPage } from "./FastLoginPage";
-export { default as SiteHeader } from "./SiteHeader";
+export { default as SiteHeader } from "../orgs/SiteHeader";
```
Comment (renanpontez) [inline src/components/mols/index.ts:9]: Why an **organism** is being export in **mols/index**?

File: src/components/atoms/Navigation/NavMobile.tsx (hunk 1 above)
Comment (femacedogomes) [reply src/components/atoms/Navigation/NavMobile.tsx:31]: we are not using in the application. Its a fake data with products variants

File: src/components/mols/index.ts (hunk 2 above)
Comment (femacedogomes) [reply src/components/mols/index.ts:9]: i think this brench is not the last one

Comment (femacedogomes) [reply src/components/mols/index.ts:9]: in my brench theres no ../orgs""

File: src/components/atoms/Navigation/NavMobile.tsx (hunk 1 above)
Comment (renanpontez) [reply src/components/atoms/Navigation/NavMobile.tsx]: so, will we need the <ul> tag for what? why we dont remove it also?

**Comment by renanpontez** [comment]
fix build

File: src/components/atoms/Navigation/NavMobile.tsx (hunk 1 above)
Comment (femacedogomes) [reply src/components/atoms/Navigation/NavMobile.tsx]: thats my mistake. i forgot to remove the html tag

File: src/components/mols/index.ts (hunk 2 above)
Comment (renanpontez) [reply src/components/mols/index.ts]: Why this conversation was resolved? Is this export correct?

**Pull request merged**